### PR TITLE
SRC-1674 integrate latest changes

### PR DESCRIPTION
**Pull request by AndriyPt**
Ready for review

### CODE REVIEW

File: docs/user_guide/1_introduction.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+</li>
+<li>Maxon DC Motors</li>
+<li>Backdrivable Gearbox</li>
+<li>Non-slip surfaces for Grasp Stability</li>
+<li>2.7 kg weight</li>
+<li>2 kg Payload (using power grasp)</li>
+<li>20 N intermittent force per fingertip, 10 N continuous</li>
+<li>260 mm maximum span</li>
+<li>1 kHz Position Control Loop , with 0.02° position sensing resolution</li>
+<li>1.39 mNm resolution direct Torque sensing at each joint</li>
```
Comment (dg-shadow) [inline docs/user_guide/1_introduction.md]: In theory maybe... Should we add "up to" or something to cover our ar...behinds?

Comment (AndriyPt) [reply docs/user_guide/1_introduction.md]: @RobShadow what do you think?

Comment (RobShadow) [reply docs/user_guide/1_introduction.md]: 10N has definitely been shown, the "intermittent" is the covering term :)

Comment (dg-shadow) [reply docs/user_guide/1_introduction.md]: Ok, I withdraw my comment.

**Pull request merged**